drivers/storage/portworx/components.go

```diff
@@ -377,6 +396,230 @@ func createVolumePlacementStrategyCRD() error {
+		"params/shared":            "Flag to create a globally shared namespace volume which can be used by multiple pods: true|false",
+		"params/priority_io":       "IO Priority: low|medium|high",
+		"params/io_profile":        "IO Profile can be used to override the I/O algorithm Portworx uses for the volumes: db|sequential|random|cms",
+		"params/group":             "The group a volume should belong to. Portworx will restrict replication sets of volumes of the same group on different nodes. If the force group option 'fg' is set to true, the volume group rule will be strictly enforced. By default, it's not strictly enforced",
+		"params/fg":                "This option enforces volume group policy. If a volume belonging to a group cannot find nodes for it's replication sets which don't have other volumes of same group, the volume creation will fail",
+		"params/label":             "List of comma-separated name=value pairs to apply to the Portworx volume",
+		"params/nodes":             "Comma-separated Portworx Node ID's to use for replication sets of the volume",
+		"params/aggregation_level": "Specifies the number of replication sets the volume can be aggregated from",
+		"params/sticky":            "Flag to create sticky volumes that cannot be deleted until the flag is disabled",
+		"params/journal":           "Flag to indicate if you want to use journal device for the volume's metadata. This will use the journal device that you used when installing Portworx. It is recommended to use a journal device to absorb PX metadata writes",
+	}
+
+	pxDb := &storagev1.StorageClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            pxDbStorageClass,
+			OwnerReferences: []metav1.OwnerReference{*ownerRef},
+			Annotations:     docAnnotations,
+		},
+		Provisioner: portworxProvisioner,
+		Parameters: map[string]string{
+			"repl":       "3",
+			"io_profile": "db",
+		},
+	}
+	if err := k8sutil.CreateStorageClass(p.k8sClient, pxDb); err != nil {
+		return err
+	}
+
+	pxDbEncrypted := &storagev1.StorageClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            pxDbEncryptedStorageClass,
+			OwnerReferences: []metav1.OwnerReference{*ownerRef},
+			Annotations: map[string]string{
+				"params/note": "Ensure that you have a cluster-wide secret created in the configured secrets provider",
+			},
+		},
+		Provisioner: portworxProvisioner,
+		Parameters: map[string]string{
+			"repl":       "3",
+			"io_profile": "db",
+			"secure":     "true",
+		},
+	}
+	if err := k8sutil.CreateStorageClass(p.k8sClient, pxDbEncrypted); err != nil {
+		return err
+	}
+
+	pxNosql := &storagev1.StorageClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            pxNosqlStorageClass,
+			OwnerReferences: []metav1.OwnerReference{*ownerRef},
+		},
+		Provisioner: portworxProvisioner,
+		Parameters: map[string]string{
+			"repl":       "2",
+			"io_profile": "db",
+		},
+	}
+	if err := k8sutil.CreateStorageClass(p.k8sClient, pxNosql); err != nil {
```

Comment:
nit: you could create a list of storage classes at the top and then just iterate through them to call CreateStorageClass. No strong preference.